checker/jdk/nullness/src/java/util/concurrent/ConcurrentLinkedDeque.java

```diff
@@ -1101,7 +1101,7 @@ public boolean removeLastOccurrence(Object o) {
      * @param o element whose presence in this deque is to be tested
      * @return {@code true} if this deque contains the specified element
      */
-    @Pure public boolean contains(@Nullable Object o) {
+    @Pure public boolean contains(Object o) {
```

Comment:
Docs and testing suggest that the parameter to this `contains` can be `null`. Note that that's _not_ true of `remove` below, so definitely keep that one unchanged, as you currently have it.

This does raise an additional question -- whether it makes sense to put `@Nullable` here when `contains(null)` would always return safely but only by returning `false`. The advantage would be for a user who happens to have, say, a `@Nullable E` to pass to this method. Supporting such users is much the reason that Java [made `contains` accept an `Object`](https://errorprone.info/bugpattern/CollectionIncompatibleType#footnote-would-requiring-e-have-been-better) in the first place, rather than an `E`. But I'm not sure if it's best to emulate that or not.